x-pack/plugin/shutdown/src/internalClusterTest/java/org/elasticsearch/xpack/shutdown/NodeShutdownShardsIT.java

```diff
@@ -88,11 +87,12 @@ public Settings onNodeStopped(String nodeName) throws Exception {
         });
 
         internalCluster().stopNode(nodeToRestartName);
+        putNodeShutdown(nodeToRestartId, SingleNodeShutdownMetadata.Type.REMOVE, null);
```

Comment:
Why do we need to have a call to `putNodeShutdown` here and in the `onNodeStopped` callback? Aren't we just shutting it down once?

Reply:
Oops, that was not intended

Reply:
Removed in [eb7e612](https://github.com/elastic/elasticsearch/pull/106502/commits/eb7e6123c12383e895eb16f4e1ad9c4c8a85472e)